.github/workflows/launchdarkly-code-references.yml

```diff
@@ -0,0 +1,25 @@
+name: Find LaunchDarkly flag code references
+
+on:
+  push:
+    branches:
+      - 'main'
+
+# cancel in-flight workflow run if another push was triggered
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  launchDarklyCodeReferences:
+    name: LaunchDarkly Code References
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 11 # This value must be set if the lookback configuration option is not disabled for find-code-references. Read more: https://github.com/launchdarkly/ld-find-code-refs#searching-for-unused-flags-extinctions
```

Comment:
If this action is running for every commit to `main`, should it only need one commit?

Reply:
Fair point... LD's documentation recommended these values. I wonder if it's caused by the fact that it cancels ongoing jobs on new commits 🤷

Reply:
We can see how it performs and make adjustments. I worry about the `fetch-depth` and the size of the repo. Possibly not cancelling the jobs depending on how long it takes as well.